dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -33,6 +34,7 @@
  *
  * @author dyuen
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
When editing a tool tag, workingDirectory is unrecognized when deserializing Tag.
When editing a workflow tag, workingDirectory is unrecognized when deserializing WorkflowVersions.
There are many others.  Not sure what's changed.